spk/syncthing/Makefile

```diff
@@ -1,20 +1,20 @@
 SPK_NAME = syncthing
-SPK_VERS = 0.12.21
-SPK_REV = 8
+SPK_VERS = 0.13.4
+SPK_REV = 9
 SPK_ICON = src/syncthing.png
 DSM_UI_DIR = app
 
 UNSUPPORTED_ARCHS = $(PPC_ARCHES)
 
-DEPENDS  = cross/busybox cross/$(SPK_NAME)
+DEPENDS  = cross/$(SPK_NAME) cross/busybox
```

Comment:
I didn't compile your PR as-is, but with my other comments, I had no problems with reverting this.